lib/store.js

```diff
@@ -47,44 +47,44 @@ var callbackFn = function(d, err, result) {
     }
 };
 
-var insert = function(collection, d, objects) {
-    collection.insert(objects, function(err, result) {
+var insert = function(col, d, objects) {
```

Comment:
I prefer this being called collection; col makes me think column in a database context